### PR TITLE
Factory Tech CT documentation

### PR DESCRIPTION
**Pull request by dalapo**

### CODE REVIEW

**Comment by jaredlll08** [comment]
@dalapo please update the PR to what @EmosewaGamer said.

File: docs/Mods/FactoryTech/CraftTweaker_Support/Drill_Grinder.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Drill Grinder
+The Drill Grinder package can be used to add recipes to Factory Tech's Chop Saw.
```
Comment (kindlich) [inline docs/Mods/FactoryTech/CraftTweaker_Support/Drill_Grinder.md]: Is it?

File: docs/Mods/FactoryTech/CraftTweaker_Support/Electric_Furnace.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Electric Furnace
+The Electric Furnace package can be used to add recipes to Factory Tech's Electric Furnace that cannot be smelted in a regular furnace.
```
Comment (kindlich) [inline docs/Mods/FactoryTech/CraftTweaker_Support/Electric_Furnace.md]: The furnace cannot be smelted in a regular furnace?

File: docs/Mods/FactoryTech/Machines.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Chop Saw, Drill Grinder, and other simple single input / single output machines. First argument is output, second is input. To remove a recipe, specify the output.
```
Comment (kindlich) [inline docs/Mods/FactoryTech/Machines.md]: You don't need this file anymore

**Comment by kindlich** [comment]
@dalapo Is this PR still a thing?
If so please change your PR to match the requested changes

**Comment by jaredlll08** [comment]
Closing because of no response from up for over 6 months...

**Comment by kindlich** [comment]
Closing this due to inactivity

**Pull request closed without merging**